CHANGES

```diff
@@ -1,5 +1,7 @@
 0.173
 - Update to Jetty 9.4.12.v20180830.
+- Update the default value of the "http-server.https.excluded-cipher" config
+  property to match Jetty's default excluded cipher suites.
```

Comment:
I think we should be more explicit here about the implications: 

All non-FS ciphers are now excluded by default. In particular, this means two things: 

1. All ciphers that uses RSA as the key exchange algorithm
2. Clients are required to support TLS1.2. Those using TLS1.0 or TLS1.1 will not establish a connection successfully.